bookx/settings.py

```diff
@@ -144,4 +144,7 @@
 # this is a bit confusing because it is not the url
 # but the name used for reverse lookup
 LOGIN_REDIRECT_URL = 'home'
-LOGOUT_REDIRECT_URL = 'home'
\ No newline at end of file
+LOGOUT_REDIRECT_URL = 'home'
+
+# custom setting by kd
+UIKIT_ROOT = os.path.join(BASE_DIR, 'static/vendor/uikit-3.0.0-rc.20/')
```

Comment:
Is there a reason for this setting?

Reply:
Mmh I think I might have put it there so I can us it in a template or something, but I can't remember what the exact reason was!